apps/nrm/models/submission.js

```diff
@@ -0,0 +1,183 @@
+  response['Customer.DualNationality'] = data['pv-nationality-second'];
+  // response['Customer.Country'];
+  // response['Customer.ReferenceNumbers'] = data['pv-ho-reference-type'];
+  response['Customer.InterpreterNeeded'] = data['pv-interpreter-requirements'] === 'yes' ? 'Yes' : 'No';
+  response['Customer.InterpreterLanguage'] = data['pv-interpreter-requirements-language'];
+  response['Customer.HelpWithCommunication'] = data['pv-other-help-with-communication'] === 'yes' ? 'Yes' : 'No';
+  response['Customer.CommunicationAid'] = data['pv-other-help-with-communication-aid'];
+  // response['Customer.PassportNumber'];
+
+  response.AdultOrChild = data['pv-under-age'] === 'yes' ? 'Child' : 'Adult';
+  response.AdultOrChildDuringExploitation = data['pv-under-age-at-time-of-exploitation'] === 'yes' ? 'Child' : 'Adult';
+  response.VictimAccount = data['what-happened'];
+  response.ExploitationLocationPresented = data['where-exploitation-happened'].toUpperCase();
+
+  response.City1 = data['where-exploitation-happened-uk-city-1'];
+  response.City2 = data['where-exploitation-happened-uk-city-2'];
+  response.City3 = data['where-exploitation-happened-uk-city-3'];
+  response.City4 = data['where-exploitation-happened-uk-city-4'];
+  response.City5 = data['where-exploitation-happened-uk-city-5'];
+  response.City6 = data['where-exploitation-happened-uk-city-6'];
+  response.City7 = data['where-exploitation-happened-uk-city-7'];
+  response.City8 = data['where-exploitation-happened-uk-city-8'];
+  response.City9 = data['where-exploitation-happened-uk-city-9'];
+  response.City10 = data['where-exploitation-happened-uk-city-10'];
+
+  // response.Region1;
+  // response.Region2;
+  // response.Region3;
+  // response.Region4;
+  // response.Region5;
+  // response.Region6;
+  // response.Region7;
+  // response.Region8;
+  // response.Region9;
+  // response.Region10;
+
+  response.ExploitationUKAddress = data['where-exploitation-happened-other-uk-other-location'];
+
+  response.ExploitationOverseasRegion = data['where-exploitation-happened-overseas-country-1'];
+  response.ExploitationOverseasRegion2 = data['where-exploitation-happened-overseas-country-2'];
+  response.ExploitationOverseasRegion3 = data['where-exploitation-happened-overseas-country-3'];
+  response.ExploitationOverseasRegion4 = data['where-exploitation-happened-overseas-country-4'];
+  response.ExploitationOverseasRegion5 = data['where-exploitation-happened-overseas-country-5'];
+  response.ExploitationOverseasRegion6 = data['where-exploitation-happened-overseas-country-6'];
+  response.ExploitationOverseasRegion7 = data['where-exploitation-happened-overseas-country-7'];
+  response.ExploitationOverseasRegion8 = data['where-exploitation-happened-overseas-country-8'];
+  response.ExploitationOverseasRegion9 = data['where-exploitation-happened-overseas-country-9'];
+  response.ExploitationOverseasRegion10 = data['where-exploitation-happened-overseas-country-10'];
```

Comment:
can this be turned into a loop?

Reply:
yes but for something like this it's better to be long winded and clear.